src/xo-app/vm/tab-advanced.js

```diff
@@ -175,8 +236,35 @@ class CoresPerSocket extends Component {
   }
 }
 
-export default ({
+export default connectStore(() => {
+  const getVgpus = createGetObjectsOfType('VGPU').pick(
```

Comment:
Please update both `xo-web` and `xo-server` to use `vgpu` instead of `VGPU` (let's use cameCase everywhere).